network/network_windows.go

```diff
@@ -318,6 +325,47 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *NetworkInfo, extIf *extern
 	return hcnNetwork, nil
 }
 
+func (nm *networkManager) addIPv6DefaultRoute() error {
+	// add ipv6 default route if it does not exist in dualstack overlay windows node from persistentstore
+	cmd := `((Get-NetIPInterface | where InterfaceAlias  -Like "vEthernet*").IfIndex)[0]`
+	ifIndex, err := nm.plClient.ExecutePowershellCommand(cmd)
+	if err != nil {
+		return errors.Wrapf(err, "error while executing powershell command to get ipv6 Hyper-V interface")
+	}
```

Comment:
what is this check for?

Reply:
for adding ipv6 route, we need to find interface index; so this is to check whether it's successful or not to get the interface index